crates/array/src/builder/fixed_size_binary.rs

```diff
@@ -113,3 +113,9 @@ impl AsSlice for FixedSizeBinaryBuilder {
         )
     }
 }
+
+impl Default for FixedSizeBinaryBuilder {
```

Comment:
`table_builder!` macro requires all fields to implement `Default` trait. we could extend `AnyBuilder` with extra arms for each amount of bytes we want to support. so instead of:
```
FixedSizeBinary(FixedSizeBinaryBuilder),
```
we would have smth like:
```
64BytesBinary(FixedSizeBinaryBuilder<64>),
32BytesBinary(FixedSizeBinaryBuilder<32>),
```
but it's kind of ugly. @kalabukdima any ideas?

Reply:
We should extend `table_builder!` macro to support simple initialization expressions.